packages/accordion/src/AccordionBody.tsx

```diff
@@ -0,0 +1,42 @@
+import { PropsWithChildren, useRef } from 'react';
+import { Box } from '@ag.ds-next/box';
+import { useElementSize, usePrefersReducedMotion } from '@ag.ds-next/core';
+import { useSpring, animated } from 'react-spring';
+
+export type AccordionBodyProps = {
+	isOpen?: boolean;
+	id: string;
+	ariaLabelledBy: string;
+	noPadding?: boolean;
+};
+
+export const AccordionBody = ({
+	ariaLabelledBy,
+	children,
+	isOpen,
+	id,
+	noPadding,
+}: PropsWithChildren<AccordionBodyProps>) => {
+	const ref = useRef<HTMLDivElement>(null);
+	const { height } = useElementSize(ref);
+
+	const prefersReducedMotion = usePrefersReducedMotion();
+	const style = useSpring({
+		from: { height: 0 },
+		to: { height: isOpen ? height : 0 },
+		immediate: prefersReducedMotion,
+	});
+
+	return (
+		<animated.section
+			id={id}
+			aria-labelledby={ariaLabelledBy}
+			role="region"
+			style={{ overflow: 'hidden', ...style }}
+		>
+			<Box padding={noPadding ? 0 : 1} ref={ref}>
```

Comment:
I wonder if it would be better to remove the `noPadidng` prop and just render a basic `div` here with no styles. 

We could create another component `AccordionBodyContent` (feel free to rename that) which is essentially just `<Box padding={1} />`.